float32 temperature
uint32 sample_timestamp
uint32 sync_timestamp
geometry_msgs/Accel accel
yesense_imu/YesenseImuQuaternion quaternion
yesense_imu/YesenseImuEulerAngle eulerAngle
yesense_imu/YesenseImuLocation location
yesense_imu/YesenseImuStatus status
yesense_imu/YesenseImuGnssData gnss
yesense_imu/YesenseImuGpsData gps